wiki/Beatmap/Pattern/Mania/Long Notes/en.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - jacks
+---
+
+# Long Notes
+
+Also referred to in-game as holds, **long notes** are notes that require you to hold and release them afterwards. They are often referred to as LNs for short and are used to represent prolonged sounds in the music.
+
+![](img/ln.png)
+
+**Releases** are known to be their own term. It refers to the ending of a **long note**, the part that requires you to release the key.
```

Comment:
I would maybe add:

`This term is frequently used because osu! takes release timing in consideration when awarding a judgement for the full long note`

Or something that is redacted better.

Reply:
```suggestion
**Releases** refers to the ending of a **long note**, the part that requires you to release the key. They are known to have their own term due to release timing in osu!mania being taken into consideration when awarding judgements for the entire long note.
```